### ISSUE TITLE
Publishing with bookstore 404s

### ISSUE THREAD

**Issue by skritch** [nteract/archived-desktop-app#4535]
**Application or Package Used**
nteract on jupyter, probably also applies to regular nteract

**Describe the bug**
The Nteract publish endpoint https://github.com/nteract/nteract/blob/aaf56557af1bbd48e834ba621815230da046bdae/packages/rx-jupyter/src/bookstore.ts#L13 404s against bookstore/master https://github.com/nteract/bookstore/blob/master/bookstore/handlers.py#L103, apparently introduced in https://github.com/nteract/bookstore/pull/134/files back in June.

**To Reproduce**
```
# install both
pip install bookstore nteract_on_jupyter
jupyter serverextension enable --py bookstore
jupyter serverextension enable --py nteract_on_jupyter

# set up the bookstore S3 config
# in ~/.jupyter/jupyter_notebook_config.py
c = get_config()
from bookstore import BookstoreContentsArchiver
c.NotebookApp.contents_manager_class = BookstoreContentsArchiver
c.BookstoreSettings.s3_bucket = your-s3-bucket
c.BookstoreSettings.workspace_prefix = your-s3-prefix
c.BookstoreSettings.published_prefix = your-s3-prefix
c.BookstoreSettings.s3_access_key_id = os.getenv('AWS_ACCESS_KEY_ID')
c.BookstoreSettings.s3_secret_access_key = os.getenv('AWS_SECRET_ACCESS_KEY')

# Run nteract
jupyter nteract
```
Create a notebook, save, publish -> see: `404 PUT /api/bookstore/published/<path>`

**Additional context**

Nteract version 6.0.1, bookstore version 2.4.2. I'm guessing I can make this work by downgrading bookstore.

--- 
I opened this here because it seems like the more correct place to fix it, but I could be wrong.

**Comment by captainsafia** [comment]
@mpacer Do you have an intuition on why this might be happening? Thanks!

**Comment by akinsella** [comment]
Last versions of Bookstore use "publish" in path instead of "published" starting version 2.2.1, this is the reason why nteract_on_jupyter for example + bookstore 2.4.1 fails to call APIs path exposed by Bookstore module.

2 options: Make the target path parameterizable in rx-jupyter or use by default new path.

**Comment by captainsafia** [comment]
@akinsella I think the most sensible fix is to go ahead and update the path to use `/publish` instead of `/published`. I can go ahead and put in a fix for this today.
